src/lib/stepper/stepper.ts

```diff
@@ -66,12 +68,17 @@ export class MatStep extends _MatStep implements ErrorStateMatcher {
 @Directive({
   selector: '[matStepper]'
 })
-export class MatStepper extends _MatStepper {
+export class MatStepper extends _MatStepper implements AfterContentInit {
   /** The list of step headers of the steps in the stepper. */
   @ViewChildren(MatStepHeader, {read: ElementRef}) _stepHeader: QueryList<ElementRef>;
 
   /** Steps that the stepper holds. */
   @ContentChildren(MatStep) _steps: QueryList<MatStep>;
+  
+  /** Mark the component for change detection whenever the content children query changes */
```

Comment:
minor nit: move this comment inside the method and use `//` style comment since it describes the reason behind some specific logic, not the purpose of the `ngAfterContentInit` method